src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(value = 0, obj = this) {
+      obj.result += value;
```

Comment:
here, you shouldn't refer to `obj`, try `this`

Reply:
fixed